test/test.js

```diff
@@ -0,0 +1,44 @@
+const Operations = artifacts.require("Operations");
```

Comment:
I think you'd better rename this file. 😊
This file tests the `addAccTab` contract function, at least the file name should be related to it.

Reply:
I will remove it. 🤫